docs/pages/installation/releases.md

```diff
@@ -13,6 +13,13 @@ A list of the [releases of Meshery](https://github.com/layer5io/meshery/releases
     <th style="white-space: nowrap;">Release Date</th>
   </thead>
   <tbody>
+       <tr>
+      <td class="centered">0.3.12</td>
+      <td>
+        <em>Mesheryctl</em> - Initial Scoop Bucket and Scoop package for mesheryctl.
```

Comment:
Can this be changed to?
**Mesheryctl - Added installation support for [Scoop CLI](https://scoop.sh) to Windows Machine**

Reply:
Nice work, gents. LGTM (either way).

If we do include a hyperlink, it would be good for that to point to our docs on how to install mesheryctl with scoop. Installation instructions and links to scoop, like homebrew, kubectl, docker, and so on are readily available on the web. If someone is going to be running emerging tech such as a service mesh, we're presumably safe in the assumption that they either are already running scoop, hence why they are looking at this release note, or are just as capable of finding it elsewhere. This is logic and that of wanting to keep Meshery installation instructions short is why I removed the Scoop installation instructions from the Quick Start guide.

Reply:
Seems quite logical @leecalcote 👍 !

Reply:
Good, good. Just a quick comment on my part. I'll leave the rest of the discussion to you (@kushthedude) and @ashiscs to finalize the release note update.

FYI- This verbiage is verbatim that of the notes in the release here - https://github.com/layer5io/meshery/releases/tag/v0.3.12. We can update there as well if you guys decide to improve the description here.

Reply:
I'll merge at end of day today, if no other comments.

Reply:
We are good to go here @leecalcote sir..